features/a.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <a>
-description: The `<a>` element TODO.
+description: The `<a>` element, creates a hyperlink to anything accessible via a URL such as external web pages, files, email addresses, locations in the same page, and more.
 spec: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element
 group: html
 compat_features:
```

Comment:
We should probably clamp this to html.elements.a to avoid having very late support dates.
Please add `compute_from: html.elements.a` and re-generate the dist file. I suppose this would make it be supported at the first version of every browser.

Reply:
I've added that, my one concern here is that there are some attributes that are only supported in one browser, or got support much later, and that won't be called out. But, saying it's not Baseline isn't the right answer either! (whee)

Reply:
I agree with your concern, and I think `<a>` is actually one of the few features for which we might want to create additional later features. The one that comes to mind is `download` because it was added substantially later and has some interesting partial support notes. There might be other.